pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1327,6 +1327,8 @@ public synchronized PulsarClient getClient() throws PulsarServerException {
                             this.getConfiguration().getBrokerClientAuthenticationPlugin(),
                             this.getConfiguration().getBrokerClientAuthenticationParameters()));
                 }
+
+                conf.setStatsIntervalSeconds(0);
```

Comment:
For the pulsar replicator, it also call this `getClient` method to init pulsar client, which should retain the stats. So we'd better add a parameter to decide whether keep the client stats.

Reply:
Why do we need the stats in replicator?

Reply:
Ok, it doesn't need the stats in replicator